.gitignore

```diff
@@ -36,4 +36,6 @@ build/
 .vscode/
 
 ### Mac OS ###
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+.env
```

Comment:
Good solution. Hoped to find this file with secrets, but didn't find :)

Well done